package.json

```diff
@@ -44,7 +44,7 @@
     "eslint-plugin-prettier": "^4.0.0",
```

Comment:
I think [`eslint-plugin-prettier@5.0.0`](https://github.com/prettier/eslint-plugin-prettier/releases/tag/v5.0.0) might also be needed for downstream projects, based on the notes in https://github.com/prettier/eslint-plugin-prettier/issues/562